pipelines/model-training.Jenkinsfile

```diff
@@ -0,0 +1,140 @@
+pipeline {
+    agent { label 'ec2orchestrator'}
+
+    environment {
+        //Input parameters
+        param_git_branch = "${params.GitBranch}"
+        param_profile = "${params.Profile}"
+        param_legion_version = "${params.LegionVersion}"
+        param_legion_infra_version = "${params.LegionInfraVersion}"
+        param_use_regression_tests = "${params.UseUnitTests}"
+        param_tests_tags = "${params.TestsTags}"
+        param_docker_repo = "${params.DockerRepo}"
+        param_debug_run = "false"
+        //Job parameters
+        sharedLibPath = "pipelines/legionPipeline.groovy"
+        commitID = null
+        cleanupContainerVersion = "latest"
+        ansibleHome =  "/opt/legion/ansible"
+        ansibleVerbose = '-v'
+        helmLocalSrc = 'false'
+        paramDexToken = "${params.DexToken}"
```

Comment:
could you please rename parameters to be the same as all others: param_This_Is_Param for those who come from parameters
and justAnInternalVar for those who are kind of pipeline constant

Reply:
Fixed